.tekton/pull-request.yaml

```diff
@@ -19,6 +19,10 @@ spec:
       value: "{{revision}}"
     - name: output-image
       value: "quay.io/redhat-appstudio/user-workload:image-controller-{{revision}}"
+    - name: hermetic
+      value: true
+    - name: prefetch-input
+      value: gomod
   pipelineRef:
     name: docker-build
     bundle: quay.io/redhat-appstudio/hacbs-core-service-templates-bundle:latest
```

Comment:
this change requires new bundle -> quay.io/redhat-appstudio-tekton-catalog/pipeline-core-services-docker-build:latest